src/code/z_collision_check.c

```diff
@@ -72,58 +115,63 @@ s32 Collider_DestroyBase(GlobalContext* globalCtx, Collider* collider) {
     return 1;
 }
 
-s32 Collider_SetBase_Actor(GlobalContext* globalCtx, Collider* collider, ColliderInit_Actor* src) {
+/**
+ * Uses default OT_TYPE1 and COLTYPE_HIT0
+ */
+s32 Collider_SetBaseToActor(GlobalContext* globalCtx, Collider* collider, ColliderInitToActor* src) {
     collider->actor = src->actor;
     collider->atFlags = src->atFlags;
     collider->acFlags = src->acFlags;
-    collider->maskA = src->maskA;
-    collider->maskB = 0x10;
+    collider->ocFlags = src->ocFlags;
+    collider->ocType = OT_TYPE1;
     collider->shape = src->shape;
     return 1;
 }
 
-// maskB = 0x10
-s32 Collider_SetBase_Set3(GlobalContext* globalCtx, Collider* collider, Actor* actor, ColliderInit_Set3* src) {
+/**
+ * Uses default OT_TYPE1
+ */
+s32 Collider_SetBaseType1(GlobalContext* globalCtx, Collider* collider, Actor* actor, ColliderInitType1* src) {
     collider->actor = actor;
-    collider->type = src->type;
+    collider->colType = src->colType;
     collider->atFlags = src->atFlags;
     collider->acFlags = src->acFlags;
-    collider->maskA = src->maskA;
-    collider->maskB = 0x10;
+    collider->ocFlags = src->ocFlags;
+    collider->ocType = OT_TYPE1;
     collider->shape = src->shape;
     return 1;
 }
 
-// maskB = src->maskB
 s32 Collider_SetBase(GlobalContext* globalCtx, Collider* collider, Actor* actor, ColliderInit* src) {
     collider->actor = actor;
-    collider->type = src->type;
+    collider->colType = src->colType;
     collider->atFlags = src->atFlags;
     collider->acFlags = src->acFlags;
-    collider->maskA = src->maskA;
-    collider->maskB = src->maskB;
+    collider->ocFlags = src->ocFlags;
+    collider->ocType = src->ocType;
     collider->shape = src->shape;
     return 1;
 }
 
-void Collider_BaseSetAT(GlobalContext* globalCtx, Collider* collider) {
+void Collider_ResetATBase(GlobalContext* globalCtx, Collider* collider) {
     collider->at = NULL;
-    collider->atFlags &= ~0x6;
+    collider->atFlags &= ~AT_HIT & ~AT_BOUNCED;
```

Comment:
Probably relevant in other places, and I don't know if we have any instances already in the repo, but this would typically be written as `~(a | b)` instead
```suggestion
    collider->atFlags &= ~(AT_HIT | AT_BOUNCED);
```